pvlib/iotools/acis.py

```diff
@@ -0,0 +1,483 @@
+import requests
+import pandas as pd
+import numpy as np
+
+
+VARIABLE_MAP = {
+    # time series names
+    'pcpn': 'precipitation',
+    'maxt': 'temp_air_max',
+    'avgt': 'temp_air_average',
+    'obst': 'temp_air_observation',
+    'mint': 'temp_air_min',
+    'snow': 'snowfall',
+    'snwd': 'snowdepth',
+
+    # metadata names
+    'lat': 'latitude',
+    'lon': 'longitude',
+    'elev': 'altitude',
```

Comment:
I guess whether to map it to elevation or altitude is arbitrary in this case? I think pvlib uses both conventions?

Reply:
I was following #1420, does it not apply here?

Reply:
Congratulations, you're paying attention! 😉It certainly does, as long as everyone agrees with the choice of altitude over elevation.